content/install-c4.md

```diff
@@ -0,0 +1,233 @@
+
+2. Run the following commands as root on the Debian systems to provision sudo, git and Ansible:
+
+```ShellSession
+# apt-get install -yq sudo git ansible
+```
+
+3. Get the Wazo Platform installer and its dependencies:
+
+```ShellSession
+# git clone https://github.com/wazo-platform/wazo-ansible.git
+# cd wazo-ansible
+# ansible-galaxy install -r requirements-postgresql.yml
+```
+
+4. Create the Ansible inventory in `inventories/c4` to specify your hosts and your preferences, as follows:
+
+Define one or more SBCs:
+
+```Ini
+[sbc_host]
+sbc_1 ansible_ssh_host=10.0.1.1 ansible_ssh_port=22
+sbc_2 ansible_ssh_host=10.0.1.1 ansible_ssh_port=22
+```
+
+Define one or more routers:
+
+```Ini
+[router_host]
+router_1 ansible_ssh_host=10.0.2.1 ansible_ssh_port=22
+router_2 ansible_ssh_host=10.0.2.2 ansible_ssh_port=22
+```
+
+Define one or more Media Proxies:
+
+```Ini
+[rtpengine_host]
+rtpengine_1 ansible_ssh_host=10.0.3.1 ansible_ssh_port=22
+rtpengine_2 ansible_ssh_host=10.0.3.2 ansible_ssh_port=22
+```
+
+Define, optionally, additional hosts for the database services:
+
+```Ini
+[database_host]
+database_1 ansible_ssh_host=10.0.4.1 ansible_ssh_port=22
+
+[redis_host]
+redis_1 ansible_ssh_host=10.0.4.2 ansible_ssh_port=22
+```
+
+Assign the roles to the aforementioned hosts assigning them to the following groups:
+
+```Ini
+[sbc:children]
+sbc_host
+
+[router:children]
+router_host
+
+[router_api:children]
+router_host
+
+[rtpengine:children]
+rtpengine_host
+
+[database:children]
+database_host
+
+[redis:children]
+redis_host
+
+[c4:children]
+sbc
+router
+router_api
+rtpengine
+database
+redis
+```
+
+Configure your Wazo Platform Class 4 with your deployment-specific settings, as follows:
+
+```Ini
+[database:vars]
+postgresql_listen_addresses = *
+
+[router:vars]
+router_api_db_host = 10.0.4.1
+router_api_redis_host = 10.0.4.2
+router_dburl_dialog = redis://10.0.4.2:6379/2
+
+[sbc:vars]
+sbc_advertise_address = c4.wazo.cloud:5060
+sbc_dispatcher_list = "1 sip:10.0.2.1:5060 16 10\n1 sip:10.0.2.2:5060 16 10"
+sbc_dburl_dialog = redis://10.0.4.2:6379/2
+
+[sbc_1_host:vars]
+sbc_advertise_address = <PUBLIC IP ADDRESS OF THE SBC> # example: 1.2.3.4
+sbc_advertise_port = 5060
+rtpengine_public_address = 3.8.157.6
+
+[sbc_2_host:vars]
+sbc_advertise_address = <PUBLIC IP ADDRESS OF THE SBC> # example: 1.2.3.4
+sbc_advertise_port = 5060
+
+[rtpengine_1]
+rtpengine_public_address = <PUBLIC IP ADDRESS OF THE MEDIA PROXY> # example: 1.2.3.4
+
+[rtpengine_2]
+rtpengine_public_address = <PUBLIC IP ADDRESS OF THE MEDIA PROXY> # example: 1.2.3.4
+```
+
+The various variables that can be customized are described at <https://github.com/wazo-platform/wazo-ansible/blob/master/README.md#variables>.
+
+By default, Wazo Platform will install the development version. To install
+the latest stable version, activate the following settings in your inventory:
+
+```Ini
+[c4:vars]
+wazo_distribution = wazo-dev-buster
```

Comment:
are you sure that these settings install stable version ?

Reply:
@mwolff44 you are right, fixed it.